lib/backends/sdc/utils.js

```diff
@@ -29,6 +29,34 @@ function networkUuidToDockerId(uuid) {
     return (uuid + uuid).replace(/-/g, '');
 }
 
+function shortNetworkIdToUuid(id) {
```

Comment:
I think it would be helpful to add comments about what the valid inputs for this function are. Adding assertions could help to clarify/check which inputs are valid too.

For instance reading this code it wasn't clear to me whether `id` could be of any length, or was expected to have a length whose value was one of a set of values.